src/lib/components/forms/form/form-components/text-area/text-area.component.spec.ts

```diff
@@ -0,0 +1,110 @@
+import { SimpleChange } from '@angular/core';
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { FormsModule } from '@angular/forms';
+import { MatFormFieldModule } from '@angular/material/form-field';
+import { MatInputModule } from '@angular/material/input';
+import { By } from '@angular/platform-browser';
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
+import { TerraPlacementEnum } from '../../../../../helpers';
+import { MockTooltipDirective } from '../../../../../testing/mock-tooltip.directive';
+import { TextAreaComponent } from './text-area.component';
+
+// tslint:disable-next-line: max-function-line-count
+describe('TextAreaComponent', () => {
+    let component: TextAreaComponent;
+    let fixture: ComponentFixture<TextAreaComponent>;
+    let inputElement: HTMLTextAreaElement;
+    const testString: string = 'foo';
+
+    beforeEach(() => {
+        TestBed.configureTestingModule({
+            declarations: [MockTooltipDirective, TextAreaComponent],
+            imports: [FormsModule, MatFormFieldModule, MatInputModule, BrowserAnimationsModule]
```

Comment:
```suggestion
            imports: [FormsModule, MatFormFieldModule, MatInputModule, NoopAnimationsModule]
```
Pascal hatte mal gezwitschert, dass das `NoopAnimationsModule` performanter ist als das `BrowserAnimationsModule ` 😅